grc/core/blocks/block.py

```diff
@@ -667,6 +703,11 @@ def get_hash():
             for key, value in parameters.items():
                 try:
                     self.params[key].set_value(value)
+                    if evaluated:
+                        try:
+                            self.params[key]._saved_evaluated = evaluated[key]
+                        except KeyError:
+                            self.params[key]._using_saved = False
```

Comment:
This attributes of `Param` should not have a leading `_` if they're accessed from outside the class